includes/footer.php

```diff
@@ -25,23 +25,19 @@
           <img height="30px" src="/assets/img/logo/sanger-tol-logo.svg" class="hide-dark">
           <img height="30px" src="/assets/img/logo/sanger-tol-logo-darkbg.svg" class="hide-light">
         </a>
-        <small class="d-block mb-3">Making awesome workflows since 2018</small>
+        <small class="d-block mb-3">Decoding earth's diversity since 2019</small>
         <small class="d-flex mb-3">Supported by<div class="social-icons">
-            <a href="/about#czi-eoss">
-              <img src="/assets/img/contributors-colour/CZI-alt.svg" alt="CZI" class="" style="max-width: 50px">
-            </a>,
-            <a href="/about#microsoft-azure">
-              <img src="/assets/img/contributors-colour/azure.svg" alt="Microsoft Azure" style="max-width: 50px">
+            <a href="/about#wellcome">
+              <img src="/assets/img/contributors-colour/Wellcome_Trust_logo.svg" alt="Wellcome" class="" style="max-width: 50px">
```

Comment:
this is just one image for both background.

Reply:
Is that an issue? I can see it properly on both backgrounds.